### ISSUE TITLE
After posting bounty make bounty populate

### ISSUE THREAD

**Issue by kevkevinpal** [stakwork/sphinx-tribes#376]
https://replit.com/join/arvztdofzp-sphinx1
if you add a bounty now it doesn't update the list of bounties till you refresh the page

**Comment by kevkevinpal** [comment]
we can probably just refresh the page after adding the bounty

**Comment by Antanasijevic** [comment]
@kevkevinpal I actually made a note about this in email i posted some time ago.
Main thing is that after bounty is created we instantly show them what they made, placed on bounty list on top.
The moment a new bounty is created, we should show some kind of preloader, clearly showing a new bounty is being created. It can be a gray box (same size as a bounty ticket) on the top, with a message like “New bounty being created. This will take a moment”. I can provide a design for it, if there is no technical limitation to make it work.
Can we do that without refreshing entire page?

**Comment by kevkevinpal** [comment]
@ecurrencyhodler what do you think about this like adding a loading animation when we create a bounty right now it kinda just pops there

**Comment by ecurrencyhodler** [comment]
Hmmm right now it's pretty fast. I don't think it's necessary. 

https://github.com/stakwork/sphinx-tribes/assets/32662508/236f920d-9ee8-4ca0-95f8-9ade59c41b44

**Comment by kevkevinpal** [comment]
its not so much for the speed but more so to show the user something was added to the top of the list, sometimes pages are loaded purposefully slow to show that something was added/done

**Comment by ecurrencyhodler** [comment]
Okay do you have an example like a video? I'm not quite sure what you're imagining.

**Comment by Antanasijevic** [comment]
Initial idea was, when new bounty is created, to show a blank bounty on the top of the list,  with spinning preloader and "Creating Bounty" label. After a second, actual text of the bounty would appear instead of placeholder, making it clear to user that new bounty is created and is on the top of the list.
Is this helpful?

![image](https://github.com/stakwork/sphinx-tribes/assets/13258550/81cd344a-1660-4772-bd78-79a6d47583bc)

**Comment by ecurrencyhodler** [comment]
Yup yes it is. We can do that. Do you have this in figma somewhere?

**Comment by ecurrencyhodler** [comment]
Okay chatted with milan. Milan agrees that we should update the UX here. But we need to factor in bounty creation from: Profile, org homepage, and homepage.

Milan also recommended that the ideal state is to have the bounties in the profile match the view in the bounties homepage. Since redesigning the homepage is on hold until further notice, we are going to push this to the backlog for now and address it when it's time.
